source/CardGames.py

```diff
@@ -161,3 +175,35 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+# simple respond for dealer in single player with known hands 
+def dealerHand(dealer, house, player):
+  houseHand = house.handSum()
+  playerHand = player.handSum()
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1   
```

Comment:
If the player and the house both have blackjack it is a push.

Reply:
Gotcha, so it's a tie